arduino/sketch/sketch.go

```diff
@@ -302,5 +302,10 @@ func GenBuildPath(sketchPath *paths.Path) *paths.Path {
 	}
 	md5SumBytes := md5.Sum([]byte(path))
 	md5Sum := strings.ToUpper(hex.EncodeToString(md5SumBytes[:]))
-	return paths.TempDir().Join("arduino", "sketch-"+md5Sum)
+
+	return getSketchesCacheDir().Join("sketch-" + md5Sum)
```

Comment:
```suggestion
	return getSketchesCacheDir().Join(md5Sum)
```

Now that they are collected under a `sketches` folder, I the `sketch-` prefix on the folder name is somewhat redundant.

I think the benefit of the folder name prefix (more clearly communicating the nature of the folder) no longer outweighs the potential harm (making output harder to read, making it more likely the user will hit the restrictive Windows command or path length limits).